### DISCUSSION TITLE
Flutter chatapp using pocketbase

### DISCUSSION THREAD

**Discussion by rohitsangwan01** [pocketbase/pocketbase#2579] [Show and tell]
Thanks for this project, i was just playing around with pocketBase and flutter , and thought of building a chatApp,  its really responsive and super fast

I guess it would be really helpful for others who wants to test or start with pocketBase and flutter, so it would be really great if we can mention this in `examples` or somewhere to easily discovered be other people

Here is link of my repo : https://github.com/rohitsangwan01/flutter_pocketbase_chat

Here are few screenshots : 

<img src="https://github.com/rohitsangwan01/flutter_pocketbase_chat/assets/59526499/2740fc9f-066e-48fd-a4eb-739c0f36823b" height="300">

<img src="https://github.com/rohitsangwan01/flutter_pocketbase_chat/assets/59526499/8d300b92-3245-434d-b9d9-de5206be2534" height="300">

<img src="https://github.com/rohitsangwan01/flutter_pocketbase_chat/assets/59526499/33fa3265-a3de-4332-ade6-dfa94dbc59b2" height="300">

<img src="https://github.com/rohitsangwan01/flutter_pocketbase_chat/assets/59526499/012be13e-c4f0-471c-a1af-b19b18414a2e" height="300">

**Comment by MehediHasannOvi** [comment]
Thnak you for share this
